mujina-idp/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM openjdk:8-jdk-alpine
+ENV SERVICE_HOST localhost
+
+VOLUME /tmp
+ADD target/laa-saml-mock-idp-1.0.0.jar app.jar
+COPY docker-idp-application.yml config/idp-application.yml
+
```

Comment:
Didn't work for me following the read me, I needed to add EXPOSE 8080 into the docker file here.

Reply:
I wonder is this a non-windows issue?

I would like to try and build/run the docker iomage/container on an Amazon linux EC2 to test it out

Reply:
Yes lets test, I did notice that the CCR docker file doesn't have the command so possibly it's just an issue with my machine